tests/API/IEXTest.cpp

```diff
@@ -0,0 +1,45 @@
+#define CATCH_CONFIG_MAIN
+#include <catch2/catch.hpp>
+#include <catch2/trompeloeil.hpp>
+
+#include "../../src/API/IEX.h"
+
+class IEX_Mock : public API::IEX
+{
+public:
+    MAKE_MOCK2(getParsedJson, json(const std::string, const bool), override);
+};
+
+TEST_CASE( "Test IEX isValidSymbol", "[API]") {
```

Comment:
This test would be clearer as three separate test cases. You've already broken it into three separate scopes and the comments you've added can be used as part of the test case name.

Reply:
I've changed to a bdd style as I think it's quite a bit clearer. Do you think it should be separate scenarios?